lib/ui/views/refund_blockchain_selection/components/refund_blockchain_list.dart

```diff
@@ -20,17 +20,33 @@ class RefundBlockchainList extends ConsumerWidget {
       getBlockchainsListProvider,
     );
     final blockchainSelectionProvider = ref.watch(
-      BlockchainSelectionFormProvider.blockchainSelectionForm,
+      blockchainSelectionFormNotifierProvider,
     );
 
     return SizedBox(
-      child: blockchains.map(
+      child: blockchains.when(
         data: (data) {
-          final filteredChains = [...data.value];
-          if (blockchainSelectionProvider.testnetIncluded == false) {
+          final filteredChains = List.of(data);
```

Comment:
Should be factorized with bridge_blockchain_list.

This can be done by creating a provider `getFilteredBlockchainListProvider`.

```dart
@riverpod
Future<List<BridgeBlockchain>> filteredBlockchainsList(
  Ref ref,
) async {
   final blockchains = await ref.watch(getBlockchainsListProvider.future);
   final archethicEnvironment = (await ref.watch(blockchainSelectionFormNotifierProvider.future)).archethicEnvironment;
   
//     if (archethicEnvironment != null) {
//            blockchains.removeWhere(
//              (element) => element.env != archethicEnvironment,
//            );
//       } else {
// ...
//          }


}

```